src/app/screens/Book/components/Row/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { View, Text, Image } from 'react-native';
+
+import defaultImage from '../../../../../assets/defaultImage.png';
+
+import styles from './styles.js';
+
+class Row extends Component {
+  render() {
+    const item = this.props;
+
+    return (
+      <View style={styles.container}>
+        {item.image_url ? (
+          <Image style={styles.image} key={item.id} source={{ uri: item.image_url }} />
+        ) : (
+          <Image key={item.id} style={styles.image} source={defaultImage} />
+        )}
```

Comment:
instead of a ternary, you could do

```
const image = item.image_url ? { uri: item.image_url } : defaultImage;
(...)
<Image key={item.id} style={styles.image} source={image} />
```

or 

```
<Image key={item.id} style={styles.image} source={ item.image_url ? { uri: item.image_url } : defaultImage } />
```